bindings/python/tests/test_async_copy.py

```diff
@@ -93,7 +94,7 @@ async def test_async_copy_nested(service_name, operator, async_operator):
     assert target_content is not None
     assert target_content == content
     await async_operator.delete(source_path)
-    await async_operator.delete(target_path)
+    await async_operator.remove_all(f'{target_path.split("/")[0]}/')
```

Comment:
Not all services support `list` operations, this may add extra requirement of capability

Reply:
In this case, maybe the solution is to new a variable and save the path of the directory? like this:
```python
target_directory = f"random_file_{str(uuid4())}/"
target_path=f"{target_directory}{str(uuid4())}/{str(uuid4())}"
...
await async_operator.remove_all(target_directory)
```

Reply:
Makes sense to me

Reply:
The core point is not about the directory issue. The `remove_all` action needs the `list` capability`. Not all of the services support it.

For me, I prefer like this way

```python
def test_demo(operator):
    path_index=[]
    path_index.append(source_path)
    try:
        for index in path_index:
            operator.remove(index)
    except:
        pass
```

We register all the path we used in index and remove it by using `remove` API

Reply:
Or you need to warp remove_all api into a `try..except` project and log out a warn message if exception has been raised